### PR TITLE
CI: Lint blog posts with Vale

### PR DESCRIPTION
**Pull request by issyl0**
- Requested in https://github.com/Homebrew/brew/pull/6826#issuecomment-564450052.
- Using the styles in Homebrew/brew, run Vale docs linting on blog posts here.
- I went for the easy approach by using a macOS runner for this as it already has Homebrew installed.

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -22,3 +22,14 @@ jobs:
...
+  vale:
+    runs-on: macos-latest
+    steps:
+    - name: Set up Git repository
+      uses: actions/checkout@master
+
+    - name: Run Vale linting on Markdown files
+      run: |
+        brew install vale
+        vale --config=$(brew --repo)/.vale.ini _posts/
```
Comment (issyl0) [inline .github/workflows/tests.yml]:
```
WARNING: Missing or invalid config file.
```

Wonderful. 🙄

**Comment by MikeMcQuaid** [comment]
Would ideally like to avoid this being two runners if there's a non-terrible way of installing on macOS? Alternatively could run all tests on macOS.

**Comment by MikeMcQuaid** [comment]
> installing on macOS

installing on Linux, sorry!

**Comment by issyl0** [comment]
Looking at [the Homebrew/brew prior art for running on Linux](https://github.com/Homebrew/brew/blob/master/.github/workflows/tests.yml#L16), it's just a copy-paste job but it still feels like more effort than this one line check probably deserves?

I'm happy to switch all of these to macOS.

**Comment by MikeMcQuaid** [comment]
> Looking at [the Homebrew/brew prior art for running on Linux](https://github.com/Homebrew/brew/blob/master/.github/workflows/tests.yml#L16), it's just a copy-paste job but it still feels like more effort than this one line check probably deserves?

Is it available in a non-Homebrew package manager so we can install it that way?

Another option would be to use the Homebrew/brew Docker image?

If neither of the above is possible: I have no strong opinions about macOS vs. the above copy-paste job.

**Comment by issyl0** [comment]
> Is it available in a non-Homebrew package manager so we can install it that way?

It is. But we need to reference the config and styles from Homebrew/brew so we need that repo anyway.

> Another option would be to use the Homebrew/brew Docker image?

I've done this.

**Comment by issyl0** [comment]
```
✔ 0 errors, 0 warnings and 0 suggestions in 15 files.
```

🎉

**Comment by MikeMcQuaid** [comment]
Great work, thanks again @issyl0!

**Pull request merged**